src/test/e2e-emulated/functions/wildcard-parameters.tact

```diff
@@ -0,0 +1,44 @@
+fun globalAdd(_: Int, _: Int): Int { return 0 }
+fun globalAddThreeWildcards(_: Int, _: Int, _: Int): Int { return 0 }
+fun globalAddMixedParams(a: Int, _: Int, b: Int): Int { return a + b }
+fun globalAddMixedParams2(a: HoldInt, _: Int, b: Int): Int { return a.val + b }
```

Comment:
I'd add one or two wildcard parameters here since the PR title says "several"

Reply:
Done :)